### PR TITLE
fixed qr code

### PR DESCRIPTION
**Pull request by rmeertens**
the qr code scanner is working again! Also: now printing the upper left corner of the code (@dewagter check this out!)

### CODE REVIEW

File: sw/airborne/modules/computer_vision/qrcode/qr_code.c [hunk 1]
```diff
@@ -99,5 +99,5 @@ struct image_t* qrscan(struct image_t *img)
   zbar_image_destroy(image);
   //zbar_image_scanner_destroy(scanner);
 
-  return NULL; // QRCode is not returning a new image.
+  return img;
```
Comment (flixr) [inline sw/airborne/modules/computer_vision/qrcode/qr_code.c]: Actually looks like return NULL is still sensible here, see `cv_run_device`...

Comment (rmeertens) [inline sw/airborne/modules/computer_vision/qrcode/qr_code.c]: See my new update ;)

**Comment by rmeertens** [comment]
Thought it would be nice to allow people to draw a rectangle around found QR codes.

**Comment by flixr** [comment]
Nice!

**Comment by dewagter** [comment]
@roland: thanks for making it compatible with dual-video!
Are you drawing in the grayscale or the YUV?

-Christophe

**Pull request merged**